TalkAloud/TalkAloud/ViewControllers/AudioRecordingsTableViewController.swift

```diff
@@ -46,11 +46,45 @@ class AudioRecordingsTableViewController: UITableViewController {
         return true
     }
     
-    override func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
-        if editingStyle == .delete {
+    override func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        let audioCell = tableView.dequeueReusableCell(withIdentifier: "audio", for: indexPath)
```

Comment:
Why are you dequing a cell here?

Reply:
I thought this was how I initialize a cell, I'm going to take that as a no here and will find another way to do it.

Reply:
Are you just wondering about the placement of that cell declaration?